### ISSUE TITLE
Support for https://docs.aws.amazon.com/AmazonS3/latest/API/RESTObjectPOST.html ?

### ISSUE THREAD

**Issue by allanlegalstart** [jamhall/s3rver#418]
As we now have multipart upload in this project, would it be hard to support this api ? (i.e would it be something a noob like me should  be able to implement ? )

**Comment by kherock** [comment]
This is in the works! Things have been at a standstill for a little while since I've been pretty busy. I haven't looked too deeply into this API yet, so you're welcome to take a look. S3rver v2 will be deprecated pretty soon, so you should be targeting the `next` branch if you want to take a look at potentially implementing.

The codebase has grown to be somewhat complex, so just to point you in the right direction the implementation would add
```
routes.js ------------------------> controllers/object.js
^                                   ^
add handler for "undefined" case    actual implementation of Object POST
in .post("/:bucket/:key+", ...)
```
 I'd recommend going with [multiparty](https://www.npmjs.com/package/multiparty) for parsing multipart data.

**Comment by allanlegalstart** [comment]
it would be `/:bucket/`  without `:key+` right ? to emulate s3 behaviour  (at least that what i observe in my network panel) 

thanks for the pointers I will try to have a look at it :)

**Comment by kherock** [comment]
You're right, sorry about that.

**Comment by allanlegalstart** [comment]
I've created a pull request #422

**Comment by kherock** [comment]
Resolved in #422.
